components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Extensions/WriterGroupModelEx.cs

```diff
@@ -20,8 +20,8 @@ public static class WriterGroupModelEx {
         public static IWriterGroupConfig ToWriterGroupJobConfiguration(
             this WriterGroupJobModel model, string publisherId) {
             return new WriterGroupJobConfig {
-                BatchSize = model.Engine?.BatchSize,
-                BatchTriggerInterval = model.Engine?.BatchTriggerInterval,
+                BatchSize = 1, // 2.7 disabled: model.Engine?.BatchSize,
```

Comment:
I would keep these values so we have a chance to enable batching manually from the file when needed. Defaulting EngineConfigurationModel to batch size to 1 respectively interval to null in enough to change the default behavior.

Reply:
I thought about it - however, the problem is existing jobs will have 50 and 10 seconds configured, so they will not get fixed.   This is a bit of a drastic step, but it is simpler than rewriting the jobs database when this is the end of 2.7 anyway, and most users will use 2.7.170.